src/drivers/intel/baytrail/ipc.c

```diff
@@ -157,6 +159,9 @@ void ipc_platform_do_cmd(struct sof *sof)
 	/* unmask busy interrupt */
 	shim_write(SHIM_IMRD, shim_read(SHIM_IMRD) & ~SHIM_IMRD_BUSY);
 
+	/* tell agent were done */
```

Comment:
Nitpick: I'd rather spell "we are" out if we want to avoid single quotes in comments, although I also think "/* we're */" works well too, at least emacs doesn't choke on that. There are more occurrences below.